R/plot_taxa.R

```diff
@@ -1309,23 +1332,33 @@ plot_sites <- function(data,
 
   cleaned <- flat_data %>%
     dplyr::select(
-      .data$longitude, 
```

Comment:
@sokole @RafaelRangel0 This line of code was changed from `.data$longitude` to `"longitude"`. If I recall correctly, the original code is required for building the package successfully. What is the reason for the change here?

Reply:
@clnsmth upon closer investigation, I don't think we need the `.data$` if the column names are quoted. This might be updated syntax with recent updates to tidyverse

Reply:
ah, but these columns were unquoted in the plotting call. I fixed that below.